src/java.base/share/classes/javax/crypto/NoSuchPaddingException.java

```diff
@@ -42,7 +42,7 @@ public class NoSuchPaddingException extends GeneralSecurityException {
     private static final long serialVersionUID = -4572885201200175466L;
 
     /**
-     * Constructs a NoSuchPaddingException with no detail
+     * Constructs a <code>NoSuchPaddingException</code> with no detail
```

Comment:
Use `{@code }` for all of the javadoc in this class?

Reply:
using `{@code }`